index 17748219e8..8d604bdad3 100644
--- a/gpio-button-hotplug.c
+++ b/gpio-button-hotplug.c
@@ -539,9 +539,8 @@ static int gpio_keys_button_probe(struct platform_device *pdev,
			struct device_node *child =
				of_get_next_child(dev->of_node, prev);

-			bdata->gpiod = devm_fwnode_gpiod_get(dev,
-				of_fwnode_handle(child), NULL, GPIOD_IN,
-				desc);
+			bdata->gpiod = devm_gpiod_get_from_of_node(dev,
+				child, "gpois", 0, GPIOD_IN, desc);

			prev = child;
		}